debian/README.Debian

```diff
@@ -0,0 +1 @@
+nvdtools for Debian
```

Comment:
I'd rather copy the short description from README.md here. Also, this `for Debian` bit feels a bit confusing to me, like if this project is only available for Debian.

Reply:
That's true. Thanks :)